### PR TITLE
Update docker source images

### PR DESCRIPTION
**Pull request by NicoLaval**

### CODE REVIEW

**Comment by NicoLaval** [comment]
Tested locally with Olympics

File: docker/server.Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
-ARG QLEVER_VERSION="0.5.6"
+ARG QLEVER_VERSION="0.5.7"
 
-FROM index.docker.io/adfreiburg/qlever:commit-b70df93
+# 10/10/24
```
Comment (ludovicm67) [inline docker/server.Dockerfile:3]:
```suggestion
# Version from: 10/10/2024
```

File: docker/ui.Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ARG QLEVER_VERSION="0.5.6"
```
Comment (ludovicm67) [inline docker/ui.Dockerfile:1]: Let's also upgrade the CLI here:

```suggestion
ARG QLEVER_VERSION="0.5.7"
```

**Review by ludovicm67 — COMMENTED** [review]
Thank you @NicoLaval for the PR!
I left some suggestions, could you take a look?

File: docker/ui.Dockerfile [hunk 3]
```diff
@@ -1,6 +1,7 @@
-ARG QLEVER_VERSION="0.5.6"
+ARG QLEVER_VERSION="0.5.7"
 
-FROM index.docker.io/adfreiburg/qlever-ui:latest@sha256:322218cb7e3190ba88f6e8448f32394a442e917ce77b836c555b833e9943a864
+# Version from: 10/06/2024
```
Comment (ludovicm67) [inline docker/ui.Dockerfile:3]:
```suggestion
# Version from: 10/10/2024
```

Comment (NicoLaval) [reply docker/ui.Dockerfile:3]: [Image](https://hub.docker.com/layers/adfreiburg/qlever-ui/latest/images/sha256-07c5ee4db5a09786b67b2fd8b98a8ab397fb88641a26bb2f43e1cbfa1980f240?context=explore) has been published on 10/06/24

Comment (ludovicm67) [reply docker/ui.Dockerfile:3]: So then the following would be better:

```suggestion
# Version from: 06/10/2024
```

**Review by ludovicm67 — APPROVED** [review]

**Pull request merged**